From: YOKOTA Hiroshi <yokota.hgml@gmail.com>
Date: Thu, 29 Oct 2026 08:35:14 +0900
Subject: Skip fragile test on hppa

Forwarded: not-needed

on hppa:
> test_line_numbers (test.basic.BasicTests.test_line_numbers) ... qemu: uncaught target signal 11 (Segmentation fault) - core dumped
> Segmentation fault
---
 test/basic.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/test/basic.py b/test/basic.py
index b0ef237..a054925 100644
--- a/test/basic.py
+++ b/test/basic.py
@@ -6,6 +6,8 @@ from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
 import codecs
+import os
+import unittest
 
 from lxml import etree
 
@@ -85,6 +87,7 @@ class BasicTests(TestCase):
             '<html xmlns="http://www.w3.org/1999/xhtml"><head><title/>'
             '<title>t</title><link href="h"/></head><body/></html>')
 
+    @unittest.skipIf(os.environ.get("DEB_HOST_ARCH") == "hppa", "skip fragile test")
     def test_line_numbers(self):
         root = parse('<html>\n<head>\n<body>\n<p><span>', line_number_attr='ln')
         self.ae(root.sourceline, 1)
